### PR TITLE
This will fix the CI tests

### PR DESCRIPTION
**Pull request by lukasedw**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -9,8 +9,11 @@ def travis?
   ENV['travis']
 end
 
+require 'chromedriver/helper'
 require 'selenium-webdriver'
 
+Selenium::WebDriver::Chrome.driver_path=Gem.bin_path("chromedriver-helper","chromedriver-helper")
```
[the remark this answers is not in the document]
Comment (lukasedw) [reply spec/spec_helper.rb]: Yep, and can i change the version of RVM on CI?, you are setting on rubuy-version 2.4.1 but there is 2.4.2...

**Comment by Hirurg103** [comment]
@lukasedw I didn't get what is wrong with ruby version on CI. Could you please explain?

**Comment by lukasedw** [comment]
> @lukasedw I didn't get what is wrong with ruby version on CI. Could you please explain?

Just to set the correct version of ruby :)

File: spec/spec_helper.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (Hirurg103) [reply spec/spec_helper.rb]: @lukasedw what is wrong with [ruby 2.4.2](https://www.ruby-lang.org/en/news/2017/09/14/ruby-2-4-2-released/)? Is it missing on Travis?

**Comment by Hirurg103** [comment]
@lukasedw what is wrong with [ruby 2.4.2](https://www.ruby-lang.org/en/news/2017/09/14/ruby-2-4-2-released/)? Is it missing on Travis?

**Comment by lukasedw** [comment]
The ruby-version was in 2.4.2, so the travis file should be with the same version, right?

**Comment by Hirurg103** [comment]
Hi @lukasedw , ah, got you! But it doesn't matters too much, because `2.4.1` and `2.4.2` differ by tiny ruby version. `2.4.2` doesn't introduce significant changes comparing to `2.4.1`, only small improvements and bug fixes

**Pull request merged**